### PR TITLE
add env to mirror configs, overriding dynconf

### PR DESCRIPTION
**Pull request by serprex**

### CODE REVIEW

**Comment by heavycrystal** [comment]
wondering if we can use https://pkg.go.dev/context#WithValue at the top level and then read at the env calls

**Comment by serprex** [comment]
> wondering if we can use https://pkg.go.dev/context#WithValue at the top level and then read at the env calls

we could, but I prefer having explicit data flow, context will be prone to breaking in weird ways

File: ui/app/mirrors/create/helpers/cdc.ts [hunk 1]
```diff
@@ -227,4 +227,18 @@ export const cdcSettings: MirrorSetting[] = [
     tips: 'Disables columns like synced_at, soft_delete etc. from being added to the destination table',
     advanced: AdvancedSettingType.ALL,
   },
+  {
+    label: 'Settings override',
```
Comment (heavycrystal) [inline ui/app/mirrors/create/helpers/cdc.ts:231]: should be more clear that it's JSON input here

**Review by heavycrystal — APPROVED** [review]
the diff hurts but you gotta do what you gotta do

**Comment by Amogh-Bharadwaj** [comment]
Documentation on usage here please :)

**Pull request merged**